modules/internal/ChapelIO.chpl

```diff
@@ -200,8 +200,8 @@ appropriately before the elements can be read.
 module ChapelIO {
   use ChapelBase; // for uint().
   use ChapelLocale;
-  use SysBasic;
-  use SysError;
+  public use SysBasic;
+  public use SysError;
```

Comment:
Is this part of the future work described in the PR message? Should these get some sort of TODO comment?
I would imagine that we don't want these to be available by default.

Reply:
Yeah, the list of public uses that I think should be investigated and considered to be retired are (ironically?) all of the non-commented ones, because I was making sure those that were somewhat well-motivated had a comment indicating that.  That said, some of the remaining ones are more problematic (expose things that probably shouldn't be by default) while others are more puzzling (why would this need to be public here?).  My list is:

seem problematic:
```
ChapelIO.chpl:  public use SysBasic;
ChapelIO.chpl:  public use SysError;
ChapelSerializedBroadcast.chpl:  public use CPtr;
```

seem like they should be unnecessary / could be cleaned up:
```
ChapelArray.chpl:  public use ChapelBase;
ChapelBase.chpl:  public use ChapelStandard;
ChapelLocale.chpl:  public use LocaleModel;
DefaultRectangular.chpl:  public use ChapelArray;
LocaleModelHelpFlat.chpl:  public use LocaleModelHelpSetup;
LocaleModelHelpFlat.chpl:  public use LocaleModelHelpRuntime;
LocaleModelHelpSetup.chpl:  public use DefaultRectangular;
LocaleModelHelpSetup.chpl:  public use ChapelNumLocales;
localeModels/flat/LocaleModel.chpl:  public use LocaleModelHelpFlat;
localeModels/flat/LocaleModel.chpl:  public use LocaleModelHelpMem;
```

might be OK, but want to think more:

```
Bytes.chpl:  public use BytesCasts;
String.chpl:  public use StringCasts;
```

I was thinking I'd spin this off into a separate issue but could also mark them in the code if you prefer.

Reply:
OK, I've put this into https://github.com/Cray/chapel-private/issues/1184